### DISCUSSION TITLE
player: add more UI elements to the interface

### DISCUSSION THREAD

**Discussion by deeplook** [asciinema/discussions#155] [Ideas]
Looking at http://theonewolf.github.io/TermRecord/htop-static.html it might be worth thinking about adding a few more UI elements, like a slider for positioning, one for playing speed, etc. One could also think about synchronisation points indicating "chapters" like in movie clips where one could directly jump to (which might need the respective reference points in the data, to be added maybe manually as a start).

**Comment by ku1ik** [comment]
That's interesting idea. I was thinking about adding ability to auto-pause at "break points", which could be nice thing to have when demoing (conf talk, peer demo etc). Break points and chapter markers you suggest feel like quite similar things.

**Comment by bkil** [comment]
* speed https://github.com/asciinema/asciinema-player/issues/163
* idle time https://github.com/asciinema/asciinema-player/issues/113
* key presses https://github.com/asciinema/discussions/issues/152
* scrolling https://github.com/asciinema/discussions/issues/144
* chapters https://github.com/asciinema/discussions/issues/99

**Comment by ku1ik** [comment]
asciinema player v3.4.0 brings [markers](https://github.com/asciinema/asciinema-player#markers-1), which act as chapter markers, and are displayed as indicators (with label tooltip on hover) on the progress bar.

There's also related new [pauseOnMarkers](https://github.com/asciinema/asciinema-player#pauseonmarkers) option to automatically pause on each enountered marker.
